places/Austin1.md

```diff
@@ -0,0 +1,3 @@
+## Austin, TX
+### Great Places to Eat
```

Comment:
Great town! Visited often when I lived in Houston. Love to know about more places to eat!